test/mysql.autoupdate.test.js

```diff
@@ -176,4 +211,25 @@ describe('MySQL connector', function() {
       done();
     });
   });
+
+  it('"mysql.columnName" is updated with correct name on create table', function (done) {
```

Comment:
Thanks, first PR attempt on this framework. I did not run the linter. Are there instructions on running the linter for this framework I missed?